man/systemd.link.xml

```diff
@@ -345,9 +345,7 @@
       <varlistentry>
         <term><varname>BitsPerSecond=</varname></term>
         <listitem>
-          <para>The speed to set for the device, the value is rounded
-          down to the nearest Mbps. The usual suffixes K, M, G, are
-          supported and are understood to the base of 1000.</para>
+          <para>The speed to set for the device in Mbps.</para>
```

Comment:
Not sure But I think other than Mbps is supported. Looking at the man of ethtool

```
speed N
Set speed in Mb/s. ethtool with just the device name as an argument will show you the supported device speeds.

```